.github/workflows/ci.yml

```diff
@@ -90,6 +90,7 @@ jobs:
            --name station \
            --detach \
            --env FIL_WALLET_ADDRESS=0x000000000000000000000000000000000000dEaD \
+           --env PASSPHRASE=secret \
```

Comment:
A passphrase is a collection of words, while a password is any string of characters. Was it intentional do select a passphrase here?

Reply:
SSH and GPG use the term "passphrase", so I chose it here.
- https://docs.github.com/en/authentication/connecting-to-github-with-ssh/working-with-ssh-key-passphrases
- https://www.gnupg.org/gph/en/manual/c14.html

TBH, I find the env var name PASSPHRASE a bit ambiguous, so I am happy to use a different name if we can find a better one. What would you suggest?

Reply:
Cool, if they use this, then I'm all in favor.

> TBH, I find the env var name PASSPHRASE a bit ambiguous, so I am happy to use a different name if we can find a better one. What would you suggest?

Explicit would be `KEY_PASSPHRASE` or `PRIVATE_KEY_PASSPHRASE` / `PKEY_PASSPHRASE`

Reply:
We are encrypting the entire keypair now - `JSON.stringify({privateKey, publicKey})`.

Should we call the env var `KEYPAIR_PASSPHRASE` then? It's a bit long, but comparable to `FIL_WALLET_ADDRESS`.

OTOH, I can imagine we can reuse this passphrase for other things in the future, so maybe it's better to keep the generic `PASSPHRASE` name.

Reply:
Let's keep using `PASSPHRASE`.